srv/ListJobs.srv

```diff
@@ -0,0 +1,22 @@
+# SPDX-FileCopyrightText: 2024, Yaskawa America, Inc.
+# SPDX-FileCopyrightText: 2024, Delft University of Technology
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# empty request
+
+---
+
+# Result of the service invocation. Values other than one (1) signal failure.
+#
+# NOTE: future versions of this service may use a different set of result codes
+uint32 result_code
```

Comment:
Initially I had this as an `int32`, but most other service defs use the unsigned variant, so I changed it to match.